addons/mail/static/src/owl/components/chatter/chatter.xml

```diff
@@ -3,25 +3,44 @@
 
     <t t-name="mail.component.Chatter" owl="1">
         <div class="o_Chatter">
-            <t t-if="storeProps.threadLocalId">
-                <ChatterTopbar
-                    class="o_Chatter_chatterTopbar"
-                    threadLocalId="storeProps.threadLocalId"
-                    t-on-o-chatter-topbar-select-attachment="_onTopbarSelectAttachment"
-                />
+            <ChatterTopbar
+                class="o_Chatter_chatterTopbar"
+                isComposerLog="state.isComposerLog"
+                isComposerVisible="state.isComposerVisible"
+                isDisabled="!storeProps.threadLocalId"
+                threadLocalId="storeProps.threadLocalId"
```

Comment:
why not make this component aware of `chatterLocalId`?

Reply:
Because he's mean so Santa Claus won't come, too bad !

Reply:
Santa finally came (y)